antenna/ext/crashstorage_base.py

```diff
@@ -17,6 +19,25 @@ class Config:
     def __init__(self, config):
         self.config = config.with_options(self)
 
+    def _path_join(self, *paths):
+        return "/".join(paths)
+
+    def _get_raw_crash_path(self, crash_id):
+        date = get_date_from_crash_id(crash_id)
+        return self._path_join("v1", "raw_crash", date, crash_id)
+
+    def _get_dump_names_path(self, crash_id):
+        return self._path_join("v1", "dump_names", crash_id)
+
+    def _get_dump_name_path(self, crash_id, dump_name):
+        # NOTE(willkg): This is something that Socorro collector did. I'm not
+        # really sure why, but in order to maintain backwards compatability, we
+        # need to keep doing it.
+        if dump_name in (None, "", "upload_file_minidump"):
+            dump_name = "dump"
```

Comment:
At some point, we should get rid of this and switch it to saving "None" and empty string to "upload_file_minidump". That'll reduce a bunch of "well, if the dump name is *this*, then we do this thing for historical reasons that's unexpected and confusing". I don't know if switching to GCS is a good opportunity for that, but it might be.

Reply:
good enough for me, i'll remove it for gcs